tool/github.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Copyright 2020 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Fast fail the script on failures.
+set -e
+
+export JAVA_HOME=$JAVA_HOME_11_X64
+
+git clone --depth 1 https://github.com/flutter/flutter.git ../flutter
+export PATH="$PATH":`pwd`/../flutter/bin:`pwd`/../flutter/bin/cache/dart-sdk/bin
+flutter config --no-analytics
+flutter doctor
+export FLUTTER_SDK=`pwd`/../flutter
+
+if [ "$IDEA_VERSION" = "4.0" -o "$IDEA_VERSION" = "4.1" ] ; then
+
+  #Install Java 8
+  wget -O- https://apt.corretto.aws/corretto.key | sudo apt-key add -
+  sudo add-apt-repository 'deb https://apt.corretto.aws stable main'
+  sudo apt-get update; sudo apt-get install -y java-1.8.0-amazon-corretto-jdk
+  export PATH=/usr/lib/jvm/java-1.8.0-amazon-corretto/jre/bin:$PATH
+
+fi
+
+java -version
+
+# Get packages for the top-level grind script utilities
+echo "pub get `pwd`"
+pub get
+
+# Get packages for the test data.
+(cd testData/sample_tests; echo "pub get `pwd`"; pub get)
+
+# Set up the plugin tool.
+(cd tool/plugin; echo "pub get `pwd`"; pub get)
+
+if [ "$DART_BOT" = true ] ; then
+
+  # analyze the Dart code in the repo
+  echo "pub global activate tuneup"
```

Comment:
We should be able to convert this over to just `dart analyze` now (not necessarily for this PR).

Reply:
That would be more readable.